addons/website/tools.py

```diff
@@ -63,6 +66,8 @@ def MockRequest(
         website=website,
         render=lambda *a, **kw: '<MockResponse>',
     )
+    if url_root:
+        werkzeug.urls.url_join(url_root, path)
```

Comment:
:eyes:

Reply:
My bad, I rushed it :grimacing:. It's fixed now.